### PR TITLE
Add getter and setter for donation transaction ID

### PR DESCRIPTION
**Pull request by helgatheviking**

### CODE REVIEW

File: includes/donations/abstract-charitable-donation.php [hunk 1]
```diff
@@ -732,6 +732,36 @@ public function flush_campaigns_donation_cache( $donation_id ) {
...
+		 * Save the gateway's transaction ID
+		 *
+		 * @param   string   $value
+		 * @return  bool
+		 * @access  public
+		 * @since   1.4.7
+		 */
+		public function set_gateway_transaction_id( $value ) {
+			$key = '_gateway_transaction_id';
+			$value = charitable_sanitize_donation_meta( $value, $key )
```
Comment (ericnicolaas) [inline includes/donations/abstract-charitable-donation.php]: Looks like this is missing a semicolon.

File: includes/donations/abstract-charitable-donation.php [hunk 2]
```diff
@@ -732,6 +732,36 @@ public function flush_campaigns_donation_cache( $donation_id ) {
...
+
+		/**
+		 * Get the gateway's transaction ID
+		 *
+		 * @return  string
+		 * @access  public
+		 * @since   1.4.7
+		 */
+		public function get_gateway_transaction_id() {
+			if( ! isset( $this->gateway_transaction_id ){
```
Comment (ericnicolaas) [inline includes/donations/abstract-charitable-donation.php]: Minor spacing issues. Add a space after `if` and before the `{`. 
```
if ( ! isset( $this->gateway_transaction_id ) {
```

**Comment by ericnicolaas** [comment]
@helgatheviking Also, make the @since `1.4.6`. We can integrate it in the next minor release.

**Comment by helgatheviking** [comment]
🎆

**Pull request merged**